Changelog.md

```diff
@@ -1,315 +1,426 @@
 # Changelog
+
 ## master (unreleased)
-## Added
```

Comment:
Max, I see you've removed the versions because there was no such release

But I think it's a some mess with versioning 
Those changes that marked as 0.7.1 for example - already in code (https://github.com/palladium-tms/palladium-view/commit/42bb974c9d3b50c076b8acf5c0afe244b3059a51)

So that means those feature implemented, but not released - so move everything that newer than 0.6.7 - to master (unreleased) but not remove them alltogether